src/kernel/mempool_entry.h

```diff
@@ -127,6 +130,7 @@ class CTxMemPoolEntry
     }
     size_t GetTxWeight() const { return nTxWeight; }
     std::chrono::seconds GetTime() const { return std::chrono::seconds{nTime}; }
+    SteadyClock::time_point GetEntryTime() const { return m_entry_time; }
```

Comment:
> Looks like this one is only to be used internally for relay and the other one is to be used for everything else?

Yeah - `SteadyClock` times aren't comparable between runs (particularly if there was a reboot inbetween), so these timestamps are only useful "internally" in general.

Could alternatively store the value of `mempool->GetSequence()` instead of the `SteadyClock` time for a similar result.

Reply:
Updated to use the mempool sequence number rather than `SteadyClock`